app/res/xml/preferences_developer.xml

```diff
@@ -85,6 +85,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-auto-purge"
         android:title="Detect and Trigger Purge on Form Save"/>
+    <EditTextPreference
```

Comment:
I can do that, but my worry is how will we configure the UI tests to choose the file from a file manager considering different file managers every phone has!